package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "start": "grunt server --node-env=dev",
     "test": "grunt test --node-env=test",
-    "outdated": "npm outdated --depth 0",
-    "postinstall": "node scripts/gen_keys"
```

Comment:
Removing this post-install may cause problems for folks doing a vanilla `npm install` rather than using docker.